linked_list/linked_list.py

```diff
@@ -89,7 +152,15 @@ def visit(self):
     # Time Complexity: ?
     # Space Complexity: ?

```

Comment:
What do you think the space and time complexity of this is? Is the time complexity dependent on the length of the linked list? Is there any new variables being created that grow larger depending upon the input?